packages/core/admin/ee/server/controllers/workflows/index.js

```diff
@@ -21,9 +24,10 @@ module.exports = {
    */
   async findOne(ctx) {
```

Comment:
Maybe we could keep the consistency between service and controller and rename this findById too, wdyt?